packs/_source/tables/bag-of-beans.json

```
@@ -13,7 +13,7 @@
       "drawn": false,
       "documentCollection": "",
       "_id": "WxO0rXJgtL1EUztI",
-      "text": "[[5d4]] toadstools sprout. If a creature eats a toadstool, roll any die. On an odd roll, the eater must succeed on a DC 15 Constitution saving throw or take [[/r 5d6]] poison damage and become poisoned for 1 hour. On an even roll, the eater gains [[/r 5d6]] temporary hit points for 1 hour.",
+      "text": "[[5d4]] toadstools sprout. If a creature eats a toadstool, roll any die. On an odd roll, the eater must succeed on a [[/save constitution dc=15]] saving throw or take [[/damage 5d6 poison]] damage and become &amp;Reference[poisoned] for 1 hour. On an even roll, the eater gains [[/healing 5d6 temp]]{5d6 temporary hit points} for 1 hour.",
```

We been moving to deferred rolls in tables because otherwise they are evaluated when embedded, leading to different table contents every time you re-render the item.

```suggestion
      "text": "[[/r 5d4]] toadstools sprout. If a creature eats a toadstool, roll any die. On an odd roll, the eater must succeed on a [[/save constitution dc=15]] saving throw or take [[/damage 5d6 poison]] damage and become &amp;Reference[poisoned] for 1 hour. On an even roll, the eater gains [[/healing 5d6 temp]]{5d6 temporary hit points} for 1 hour.",
```